pkg/server/api/github_test.go

```diff
@@ -0,0 +1,116 @@
+package api
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http/httptest"
+	"testing"
+
+	"golang.org/x/crypto/ssh"
+
+	"github.com/sirupsen/logrus"
+	"github.com/sirupsen/logrus/hooks/test"
+	"github.com/square/sharkey/pkg/server/config"
+	"github.com/square/sharkey/pkg/server/storage"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestEmptyGitHubUser(t *testing.T) {
+	hostname := "proxy"
+	header := "X-Forwarded-User"
+	c, err := generateContext(t)
+	require.NoError(t, err)
+
+	// set auth proxy
+	c.conf.AuthenticatingProxy = &config.AuthenticatingProxy{
+		Hostname:       hostname,
+		UsernameHeader: header,
+	}
+	c.conf.GitHub.IncludeUserIdentity = true
+
+	hook := test.NewLocal(c.logger)
+
+	for i := 0; i < 5; i++ {
```

Comment:
I don't see `i` being used - why there is a loop btw.?

Reply:
I copied this from the other tests, likely just to test it a few times for consistency